### DISCUSSION TITLE
204

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#188] [General]
# 潮流周刊第204期 - 肉桂拿铁

封面图拍摄于周末自己在家做了一杯肉桂拿铁，热咖啡加上一点儿锡兰肉桂非常好喝，适合冬天有阳光的时候。

https://weekly.tw93.fun/posts/204-%E8%82%89%E6%A1%82%E6%8B%BF%E9%93%81

**Comment by ModyQyW** [comment]
坂本龙一给日料餐厅挑选的歌单链接多了一个 l，有需要的朋友也可以点这个 https://www.youtube.com/playlist?list=PLyF_H5mySFA3TqDp9uQZ0TJrjjJ_b38Ty

**Comment by JabinHao** [comment]
打卡
